styles/layouts/scss/modal.scss

```diff
@@ -0,0 +1,68 @@
+@use '../../global-styles/_fonts.scss' as *;
+@use'../../global-styles/_colors.scss' as*;
+
+.booking-modal__step{ 
+    display: none;//Initially hidden, pops up with btn
+    position: fixed; //modal is in its fixed position when scrolling
+    transform: translate(-50%, -50%); //Vertical and horisontal centering
+    background-color: white;
+    outline: 3px solid rgba(0,0,0,0.1);
+    width: 600px;
+    height: 500px;
+    top: 50%;
+    left: 50%;
+    z-index: 1000;
+}
+
+.modal__content-loading {
+    position: relative;
+    color:#444444;
+    font-weight: 400;
+}
+
+.booking-modal__step .modal__content .booking__date { 
+    font-weight: bolder;
+    color:#444444; 
+    
+}
+
+.booking__date-custom{
+    color:#444444;   
+    margin-top: 4rem;
+    font-weight: 600;
+}
+
+.custom__date { 
+    color: white;
+    border: 3px solid lightgray;
+    border-radius: 4px;
+    font-size: xx-large;
+}
+
+.custom__date::-webkit-calendar-picker-indicator{
+    display: none;
+}
```

Comment:
This could be why the date picker is invisible. Could you try commenting this out?

Reply:
I did not want the numbers like this XXXX XX XX - so I had to make them white. The calendar picker displayed to none was because in whimsical the input field was EMPTYYY. Now it looks good but does not look like whimsical

Reply:
for me the display is correct. :) 
![image](https://github.com/user-attachments/assets/2e26ebdf-52f6-43d2-998e-94f7aeabc167)

Reply:
you could add ateInput.addEventListener('keydown', (e) => e.preventDefault()); 
to prevent the user to type in the field and force them to do a selection instead